src/components/Login/Login.tsx

```diff
@@ -3,14 +3,23 @@ import { Formik, Form } from "formik";
 import * as Yup from "yup";
 import { toast } from "react-toastify";
 import { useTranslation } from "react-i18next";
-import { Link } from "react-router-dom";
 import { useNavigate } from "react-router-dom";
 
 import { login } from "services/auth.service";
 import { Button, Input, IconEye, IconPassword, IconText, LogoPage, IconEyeHide, Toast, Line } from "styles";
-import { LoginForm, StyledInlineErrorMessageForm, View, LabelStyle, Footer } from "./Login.style";
+import {
+  LoginForm,
+  StyledInlineErrorMessageForm,
+  View,
+  LabelStyle,
+  Footer,
+  FooterWrapperLeft,
+  LinkFooter,
+  FooterWrapperRight,
+} from "./Login.style";
 import ILogin from "./Login.interface";
 import { paths } from "config/paths";
+import Navbar from "components/Navbar/Navbar";
```

Comment:
import powinien wyglądać tak
```js
import { Navbar } from "components"
```

Reply:
tylko że jak zmieniam ścieżkę to sypie błędem

Reply:
<img width="461" alt="image" src="https://user-images.githubusercontent.com/84628957/164756972-ee530895-11b3-48ef-8af7-7c100b180e5b.png">

Reply:
fix